### DISCUSSION TITLE
We need more infos on D1's cold starts

### DISCUSSION THREAD

**Discussion by 55lj** [cloudflare/workers-sdk#3304] [Q&A]
D1 has significant cold starts after idling for a while, just check the Northwind demo. I assume this has technical reasons and/or is intended.

Would be great if we knew, from what/when/where a cold start is triggered. And, if i's the same on any pricing tier.

**Comment by WillSmithTE** [comment]
Since this is the first search result, any more info here?

**Comment by waynesbrain** [comment]
It's fast now! https://northwind.d1sql.com/suppliers

**Comment by MichaelTendoSsemwanga** [comment]
I get 15s to > 1 minute cold starts on my 3.5GB single table database, so the problem very much still exists.
